allennlp/training/trainer.py

```diff
@@ -32,7 +32,8 @@ def __init__(self,
                  cuda_device: int = -1,
                  grad_norm: Optional[float] = None,
                  grad_clipping: Optional[float] = None,
-                 log_one_line_per_batch: bool = False) -> None:
+                 log_one_line_per_batch: bool = False,
+                 validation_metric_decreases: bool = True) -> None:
```

Comment:
Can we handle this just by saying in the `validation_metric` docs that prepending a `-` means it decreases?  The rest of the logic stays the same, you just do a tiny bit of processing in `__init__` to handle this.